aion/web/elm/src/Ranking/View.elm

```diff
@@ -2,56 +2,116 @@ module Ranking.View exposing (..)
 
 import General.Models exposing (Model)
 import Html exposing (..)
+import Html.Attributes exposing(style, src, value)
 import Msgs exposing (Msg(..))
 import Bootstrap.Table as Table
 import Bootstrap.Grid as Grid
+import Bootstrap.Table exposing(rowAttr)
 import Bootstrap.Grid.Col as Col
-import Ranking.Models exposing (PlayerScore, RankingData)
+import Bootstrap.Form as Form
+import Bootstrap.Form.Select as Select
+import Ranking.Models exposing (PlayerScore, RankingData, CategoryRanking)
 import RemoteData exposing (WebData)
+import Urls exposing (host)
+import Navigation exposing (Location)
 
 
 rankingView : Model -> Html Msg
 rankingView model =
     Grid.container []
         [ Grid.row []
-            [ Grid.col [ Col.xs4 ]
-                [ h4 [] [ text "General" ] ]
+            [ Grid.col [ Col.xs4, Col.offsetXs1 ]
+                [ Form.group []
+                    [ Form.label [] [ text "Select Category" ]
+                    , Select.select [ Select.onChange OnRankingCategoryChange ] (selectCategoriesAttributes model)
+                    ]
+                ]
             ]
         , Grid.row []
-            [ Grid.col [ Col.xs4 ]
+            [ Grid.col [ Col.xs10, Col.offsetXs1 ]
                 [ rankingTable model ]
             ]
         ]
 
 
+selectCategoriesAttributes : Model -> List (Select.Item msg)
+selectCategoriesAttributes model =
+    case model.rankingData.data of
+        RemoteData.Success rankingData ->
+            List.map (\category -> Select.item [ value (toString category.categoryId) ] [ text category.categoryName] ) rankingData.rankingList
```

Comment:
I think it would work better if this lambda was exported to a named function. Mapping overt collection values with anonymous functions works great if you are doing simple changes. Here it looks a bit more complex and it would be easier to read if the function's name told you what it's doing. Like this:
```
List.map displayRankingEntry rankingData.rankingList
```